main/WebServer.cpp

```diff
@@ -3481,6 +3481,12 @@ namespace http {
 
 		void CWebServer::Cmd_DeleteDateRange(WebEmSession &session, const request &req, Json::Value &root)
 		{
+			if (session.rights != 2)
+			{
+				_log.Log(LOG_ERROR, "User: %s tried to delete a daterange!", session.username.c_str());
```

Comment:
We don't use these log messages, but only issue session.reply_status = reply::forbidden;
(Just for a very few sporadic calls there is a log message, and for the uservariable it is not needed as well)